spec/controllers/v2/concerns/with_metadata_spec.rb

```diff
@@ -19,6 +19,9 @@
 # ```
 #
 RSpec.shared_examples 'with metadata' do |*parents|
+  let(:first_offset_keys) { %w[first last next] }
+  let(:middle_offset_keys) { %w[first last next previous] }
+  let(:last_offset_keys) { %w[first last previous] }
```

Comment:
I don't see the point of extracting these if they are only used at a single location.